### ISSUE TITLE
kubectl version should print the client version before going to the server

### ISSUE THREAD

**Issue by ghodss** [kubernetes/kubernetes#13283]
`kubectl version` used to print the client version before it attempted to get the server version, which means even if the server is unavailable you can still see the client version. That seems to be broken now, and instead you just get an error when the server is unavailable:

```
$ kubectl version
error: couldn't read version from server: Get http://localhost:8080/api: dial tcp 127.0.0.1:8080: connection refused
```

Instead, the client version should print before going to check the server version (which should still print an error if the connection fails).

**Comment by feihujiang** [comment]
This is fixed by #13179

**Comment by ghodss** [comment]
Thank you!
